Tests/LibCrypto/TestCurves.cpp

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2022, stelar7 <dudedbz@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/ByteBuffer.h>
+#include <LibCrypto/Curves/X25519.h>
+#include <LibTest/TestCase.h>
+
+TEST_CASE(test_x25519)
+{
+    // https://datatracker.ietf.org/doc/html/rfc7748#section-6.1
+    u8 generator_array[32] = {
+        9
+    };
+    ReadonlyBytes generator { generator_array, 32 };
+
+    u8 server_private_key_array[32] = {
+        0x77, 0x07, 0x6d, 0x0a, 0x73, 0x18, 0xa5, 0x7d,
+        0x3c, 0x16, 0xc1, 0x72, 0x51, 0xb2, 0x66, 0x45,
+        0xdf, 0x4c, 0x2f, 0x87, 0xeb, 0xc0, 0x99, 0x2a,
+        0xb1, 0x77, 0xfb, 0xa5, 0x1d, 0xb9, 0x2c, 0x2a,
+    };
+    u8 client_private_key_array[32] = {
+        0x5d, 0xab, 0x08, 0x7e, 0x62, 0x4a, 0x8a, 0x4b,
+        0x79, 0xe1, 0x7f, 0x8b, 0x83, 0x80, 0x0e, 0xe6,
+        0x6f, 0x3b, 0xb1, 0x29, 0x26, 0x18, 0xb6, 0xfd,
+        0x1c, 0x2f, 0x8b, 0x27, 0xff, 0x88, 0xe0, 0xeb,
+    };
```

Comment:
The linter seems to be complaining about these arrays. It seems to want every number on a new line. What to do?

Reply:
`// clang-format off` beforehand and `// clang-format on` afterwards should do the trick

Reply:
Thanks!

Reply:
You can also remove the last trailing comma